apps/docs/docs/state/actions/actions.mdx

```diff
@@ -31,223 +30,142 @@ npm install --save @rx-angular/state
 yarn add @rx-angular/state
 ```
 
-## Resources
-
-**Example applications:**
-A demo application is available on [GitHub](https://github.com/BioPhoton/rx-angular-state-actions).
-
-## Motivation
-
-Actions are a fundamental part of state management and reactive systems. The @rx-angular/state library provides a powerful set method, but in certain scenarios, you might need to add specific behaviors to your user input or incoming events.
-
-Typically, Subjects are used to implement this functionality. However, in larger applications, this approach often results in code cluttered with Subjects, leading to increased complexity.
-
-That's where @rx-angular/state/actions comes in. It allows you to create observable action streams in a clean, maintainable, and less error-prone way, thus minimizing the amount of boilerplate code required.
-
-## RxAngular Actions
-
 This package helps to reduce code used to create composable action streams.
 It mostly is used in combination with state management libs to handle user interaction and backend communication.
 
-### Setup
+## Setup
 
-The coalescing features can be used directly from the `cdk` package or indirectly through the `state` package.
-To do so, install the `cdk` package and, if needed, the packages depending on it:
+`Actions` can be imported as following:
 
-1. Install `@rx-angular/state`
+```ts
+import { rxActions } from '@rx-angular/state/actions';
 
-```bash
-npm i @rx-angular/state
-# or
-yarn add @rx-angular/state
+@Component({
+  standalone: true,
+  template: `...`,
+})
+export class AnyComponent {
+  actions = rxActions<UI>();
+}
 ```
 
-### Basic usage
+### Basic usage in a Component
 
-By using RxAngular Actions we can reduce the boilerplate significantly, to do so we can start by thinking about the specific section their events and event payload types:
+# Actions
```

Comment:
```suggestion

```

why do we have this headline? Does it make sense? Looks like we can just remove it